src/FSharp.Core/fslib-extra-pervasives.fsi

```diff
@@ -15,6 +15,48 @@ module ExtraTopLevelOperators =
     open Microsoft.FSharp.Collections
     open Microsoft.FSharp.Quotations
 
+    /// <summary>Print to the console.</summary>
+    ///
+    /// <param name="text">The string to print.</param>
+    ///
+    /// <example id="print-1">Using interpolated strings:
+    /// <code lang="fsharp">
+    /// print $"Write three = {1+2}"
+    /// </code>
+    /// After evaluation the text <c>"Write three = 3"</c> is written to the console.
+    /// </example>
+    /// <example id="print-2">Using pipeline operator:
+    /// <code lang="fsharp">
+    /// "1"
+    /// |> String.replicate 4
+    /// |> print
+    /// </code>
+    /// After evaluation the text <c>"1111"</c> is written to the console.
+    /// </example>
+    [<CompiledName("Print")>]
+    val print: text: string -> unit
+
+    /// <summary>Print to the console and adds a newline.</summary>
```

Comment:
```suggestion
    /// <summary>Print to the console and add a newline.</summary>
```

All the other `*print*` functions use the bare infinitive/imperative form throughout.

Reply:
Good catch!